docs/README.md

```diff
@@ -15,30 +15,30 @@ It's a ["make"](<https://en.wikipedia.org/wiki/Make_(software)>) for NPM monorep
 ## Why Bilt
 
 (To go directly to the meat of Bilt, read
-[Structure of a Bilt monorepo](./docs/monorepo-structure.md) and then
-go to the [Getting started](./docs/getting-started.md) chapter.)
+[Structure of a Bilt monorepo](./monorepo-structure.md) and then
```

Comment:
Am I missing something here? The links in the md file work today, so why were they changed?

Reply:
The links in the md files **are** working well in the github repo homepage.

It's very minor issue but if you'll open the inner md file and navigate you'll see the broken links. For example open [this](https://github.com/giltayar/bilt/tree/main/docs) or [this](https://github.com/giltayar/bilt/blob/main/docs), and click on `Monolithic codebases vs monorepos` link.

Also, in _visual studio code_ links are not working well before this change.

BTW - thank you for this project! 🚀

Reply:
Of course. But what is important is the github repo homepage (and linked pages from there), and not the `/docs`. It's supposed to work there and only there. I'm not even sure why there's a readme in `/docs`. I think it can just be deleted.

Reply:
Deleted `/docs/README.md`

The linking issue appear also in other /docs/ files. 
For example open [this](https://github.com/giltayar/bilt/blob/main/docs/build-configurations.md) and click on `build configurations` link